crates/sui-config/Cargo.toml

```diff
@@ -20,8 +20,8 @@ multiaddr = "0.14.0"
 once_cell = "1.11.0"
 tracing = "0.1.36"
 
-narwhal-config = { git = "https://github.com/MystenLabs/narwhal", rev = "50411aa4b8b6eac7e45fa0e0da4ad8fc6c20395e", package = "config" }
-narwhal-crypto = { git = "https://github.com/MystenLabs/narwhal", rev = "50411aa4b8b6eac7e45fa0e0da4ad8fc6c20395e", package = "crypto" }
+narwhal-config = { git = "https://github.com/punwai/narwhal-1", rev = "dbecf46841ecd84aa1c7829944b743f15a471ae1", package = "config" }
```

Comment:
We should probably update the official NW upstream Vs pointing to a personal repo

Reply:
Yep, this relies on an unmerged Narwhal PR - I can only start merging this PR after https://github.com/MystenLabs/sui/pull/3586 is merged